src/sentry/options/defaults.py

```diff
@@ -398,3 +398,8 @@
 # Mechanism for dialing up the last-seen-updater, which isn't needed outside
 # of SaaS (last_seen is a marker for deleting stale customer data)
 register("sentry-metrics.last-seen-updater.accept-rate", default=0.0)
+
+# default brownout crontab for api deprecations
+register("api.deprecation.brownout-cron", default="0 12 * * *", type=String)
+# Brownout duration to be stored in ISO8601 format for durations (See https://en.wikipedia.org/wiki/ISO_8601#Durations)
+register("api.deprecation.brownout-duration", default="PT1M")
```

Comment:
While I appreciate the standardization of using these date formats here, I do think it makes for a more confusing time trying to understand the series of letters an numbers than the existing format of:

```python
timedelta(minutes=1)
```

Is it possible to require a `datetime.timedelta` object as this parameter instead of a string in this format? It'd also remove the need for an extra dependency and error handler for ISO8601Errors.

Reply:
So I talked with ops, and they don't want objects stored in option manager ( they should be able to add/modify values without a redeploy). So for blackout duration, I needed to store it as a string.